webapp/channels/src/components/analytics/true_up_review.tsx

```diff
@@ -34,7 +34,8 @@ import './true_up_review.scss';
 const TrueUpReview: React.FC = () => {
     const dispatch = useDispatch();
     const isCloud = useSelector(isCurrentLicenseCloud);
-    const isAirGapped = !useCWSAvailabilityCheck();
+    const cwsAvailability = useCWSAvailabilityCheck();
+    const isAirGapped = cwsAvailability === CSWAvailabilityCheckTypes.Unavailable;
```

Comment:
Do we need to also add here a check to `Unknown` given that now we split it those out seems like what was `false` or `undefined` can now be `Unavailable` or `Unknown`

Reply:
Or simply check if it's anything other than `Available`.

Reply:
This should have been a redux store value at the app load thats all, because customer apps dont suddenly change from being to not being air gapped in between, something to fix later. i will add the condition for now.